types/file.d.ts

```diff
@@ -1,13 +1,9 @@
-// based on https://unpkg.com/browse/formdata-node@4.0.1/@type/File.d.ts (MIT)
+// Based on https://github.com/octet-stream/form-data/blob/2d0f0dc371517444ce1f22cdde13f51995d0953a/lib/File.ts (MIT)
 /// <reference types="node" />
 
-import { Blob } from 'buffer'
+import { Blob, BlobOptions } from 'buffer'
 
-export interface FileOptions {
-  /**
-   * Returns the media type ([`MIME`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types)) of the file represented by a `File` object.
-   */
-  type?: string
+export interface FileOptions extends BlobOptions {
```

Comment:
Maybe let's name this [`FilePropertyBag`](https://w3c.github.io/FileAPI/#ref-for-dfn-FilePropertyBag)? What do you think?

Reply:
I do think it might broke someone's code if we rename it right away since it is exposed interface :)
But we can deprecate it and rename as you proposed.

Reply:
My reasoning is to follow W3C naming, but I don't have anything against `FileOptions` as well :)